documentation/Tutorials/WPF/Categories1.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: page
+title: Category Model and ViewModel
+---
+
+[Contents](Contents) [Previous](ExpensesLogbook) [Next](CategoryEditorHookup)
+
+## Create a categories user control
+
+In the setup chapter, we created an empty Shell with a placeholder for user controls. In this chapter we will create the first user control and show how to hook it up in the application.
+
+The Categories function allows to define categories for expenses and in the next chapter expenses will be assigned a category.
+
+### Create a model
+
+The first step is to create a simple model and put it into the Models folder. This is a bit of a simplified solution architecture, but let's keep it that way for tutorial purposes and avoid a lot of confusion. This is justified because it is a small application.
+
+So, create a ``CategoryModel`` class that looks like this:
+
+```csharp
+namespace Caliburn.Micro.Tutorial.Wpf.Models
+  {
+  public class CategoryModel
```

Comment:
See my comment below. I prefer not to do that in this example. There will be similar situations to demonstrate that.

Reply:
Added an explanation. I will include sample code to represent the suggested solution, because I intend to add some more user controls. One of them will use a BindableCollection.